### ISSUE TITLE
Turning off notes with highest activation 

### ISSUE THREAD

**Issue by peiherng97** [RichardYang40148/MidiNet#10]
In your report, you stated 

> For creating a monophonic note sequence,we added a layer to the end of G to turn off per time step all but the note with the highest activation." May i know how did you implement this in your code? Is it the sigmoid activation function in the last layer? 
>
`tf.nn.sigmoid(deconv2d(h4, [self.batch_size, 16, 128, self.c_dim],k_h=1, k_w=128,d_h=1, d_w=2, name='g_h5'))`

**Comment by RichardYang40148** [comment]
Hi, for generation, simply took the note with highest activation in every time step. (e.g., max index across vector)
This shouldn't include additional implementation in the activation function.
